pycipapi/cipapi_client.py

```diff
@@ -126,6 +127,14 @@ def submit_clinical_report_raw(self, payload, partner_id, analysis_type, report_
         url = self.build_url(self.url_base, self.CR_ENDPOINT, partner_id, analysis_type, report_id) + '/'
         return self.post(url, payload, params=params)
 
+    def submit_variant_interpretation_logs_raw(self, payload, case_id, case_version, **params):
+        case_id_version = "{ir_id}-{ir_version}".format(
+            ir_id=case_id,
+            ir_version=case_version
+        )
+        url = self.build_url(self.url_base, self.IR_ENDPOINT, case_id_version, 'variant-interpretation-log') + '/'
```

Comment:
is IR_ENDPOINT correct?

Reply:
Yeah, the vil endpoint sits underneath the IR endpoint 
![image](https://user-images.githubusercontent.com/20341309/116094445-718cdd00-a69f-11eb-9bca-b9b1c07468c7.png)